### ISSUE TITLE
Lootable corpses?

### ISSUE THREAD

**Issue by bdagenais2** [unldenis/Corpse#4]
Hello, The plugin you were inspired by had the ability to right click on the corpse to retrieve the items of the dead player. Is this something you plan to add to your plugin as well?

**Comment by unldenis** [comment]
Yes, I had thought about it, there is the problem that you cannot hear the clicks on a lying fake body side.
The solution could be to create hitboxes like on [Hologram-Lib](https://github.com/unldenis/Hologram-Lib).
